db/schema.rb

```diff
@@ -23,7 +23,22 @@
     t.datetime "updated_at", null: false
   end
 
-  create_table "users", force: :cascade do |t|
+  create_table "timetables", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
+    t.date "date"
+    t.string "open"
```

Comment:
I am not sure if this is worth changing now, but since we are using "closed" to represented whether or not a library is closed and "close" for the time it seems like those are a little too similar. Maybe open_time, close_time might make those columns a little clearer. Again, not sure if it's worth changing.